### PR TITLE
Add Logary Rutta to incubator/

### PR DESCRIPTION
**Pull request by haf**
#### What this PR does / why we need it:

Doing cloud-native logging, metrics and monitoring across languages and platforms is hard. Rutta solves a lot of the hardships that software architects face, trying to unify a logging/metrics pipeline.

#### Special notes for your reviewer:
This is my first PR and my first chart! 😅

#### Checklist

[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by haf** [comment]
Hey c'mon! That's not fair!

image

**Comment by haf** [comment]
What do I need to do to progress this?

**Comment by haf** [comment]
xxxxxxxx

**Comment by haf** [comment]
xxxxxxxx

**Comment by maorfr** [comment]
hey @haf 

Have you considered hosting this chart and adding it to helm hub?
https://github.com/helm/hub/blob/master/Repositories.md

**Comment by haf** [comment]
Not stale

**Comment by haf** [comment]
@maorfr No I don’t want that. Thanks for the suggestion though.

**Comment by haf** [comment]
xxxxxxxxxx

**Comment by haf** [comment]
Not

File: incubator/rutta/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+- metrics
+- structured
+- semantic
+- cep
+- prometheus
+- influx
+- stackdriver
+- rabbitmq
+maintainers:
+- name: Henrik Feldt
```
Comment (cpanato) [inline incubator/rutta/Chart.yaml:20]: should be your github handler and good practice is to add the email as well

File: incubator/rutta/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "rutta.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+
+    spec:
+      containers:
+      - name: {{ .Chart.Name | quote }}
+        image: {{ .Values.image | default "haaf/rutta:5.0.0-rc.1.597e31a3" | quote }}
```
Comment (cpanato) [inline incubator/rutta/templates/deployment.yaml:27]: you dont need to set the default image

File: incubator/rutta/templates/service.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  - port: {{ .binding.port | required "Your binding needs a port specified." }}
+    {{- if (eq .mode "udp") }}
+    protocol: "UDP"
+    {{- end }}
+    name: "{{ .mode }}-{{ .codec }}"
+  {{- end }}
+  selector:
+    app.kubernetes.io/name: {{ include "rutta.name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+{{- end }}
```
Comment (cpanato) [inline incubator/rutta/templates/service.yaml:24]: new line

File: incubator/rutta/templates/tests/test-connection.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  - name: curl
+    image: haaf/rutta-curl
+    args:
+    - '-X'
+    - 'OPTIONS'
+    - '-i'
+    - 'http://{{ $fullName }}:{{ .binding.port }}/i/logary'
+  {{ end }}
+  {{- end }}
+  restartPolicy: Never
```
Comment (cpanato) [inline incubator/rutta/templates/tests/test-connection.yaml:27]: new line

**Review by cpanato — CHANGES_REQUESTED** [review]
after you make the changes please move this to the stable folder. thanks!

**Comment by maorfr** [comment]
please take a look at https://github.com/helm/helm/pull/6095#issue-301608768

we strongly encourage people to self host chart repos:
https://github.com/helm/hub/blob/master/Repositories.md

you can even do that using GitHub pages:
https://github.com/helm/hub/blob/master/docs/github.md#github-pages-as-a-repository

**Pull request closed without merging**